boards/custom_stm32.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
This is a copy of the board file used for the nucleo build:
   ~/.platformio/platforms/ststm32/boards/nucleo_l452re.json
with the variant line removed to prevent all the Arduino libraries from being included.

I don't think there are any legal comments in a json file.  I'll add a readme file to the directory though.